rfcs/0003-timestamps-and-ttl.md

```diff
@@ -211,8 +177,11 @@ table SsTableInfo {
     // the current encoding version of the data block
     format_version: uint;
     
-    // the metadata features that are encoded with each row, in order that they are encoded
+    // the metadata features that are encoded with each row, in declaration order in MetaFeatures
```

Comment:
Not clear what declaration order means here. Do you mean that they must always be in timestamp -> TTL -> row type order?

Reply:
Yes that was the intention (it's the "canonical ordering")